backend/third_party_licenses/tekton-driver.csv

```diff
@@ -0,0 +1,110 @@
+cloud.google.com/go/compute/metadata,https://github.com/googleapis/google-cloud-go/blob/compute/metadata/v0.2.3/compute/metadata/LICENSE,Apache-2.0
```

Comment:
We can remove the all 5 `tekton-*.csv`. I forgot to remove them from the kfp-tekton branch. These license was for the old implementation where we are building driver etc as a new container task.